EasyCommands/Commands/Commands.cs

```diff
@@ -107,53 +122,50 @@ public class VariableAssignmentCommand : Command {
                 return true;
             }
 
-            protected override Command Clone() { return new VariableAssignmentCommand(variableName, variable, useReference); }
+            public override Command Clone() { return new VariableAssignmentCommand(variableName, variable, useReference); }
         }
 
         public class ControlCommand : Command {
-            Variable loopAmount = new StaticVariable(new NumberPrimitive(1));
             ControlType controlType;
             public ControlCommand(List<CommandParameter> parameters) {
                 int controlIndex = parameters.FindIndex(p => p is ControlCommandParameter);
-                int loopIndex = parameters.FindIndex(p => p is VariableCommandParameter);
                 if (controlIndex < 0) throw new Exception("Control Command must have ControlType");
                 controlType = ((ControlCommandParameter)parameters[controlIndex]).Value;
-                if (loopIndex >=0) loopAmount = ((VariableCommandParameter)parameters[controlIndex]).Value;
             }
 
-            public ControlCommand(ControlType controlType, Variable loopAmount) {
+            public ControlCommand(ControlType controlType) {
                 this.controlType = controlType;
-                this.loopAmount = loopAmount;
             }
 
             public override bool Execute() {
                 switch (controlType) {
                     case ControlType.STOP:
-                        RUNNING_COMMANDS = null; STATE = ProgramState.STOPPED; break;
-                    case ControlType.PARSE:
-                        RUNNING_COMMANDS = null; ParseCommands(); STATE = ProgramState.STOPPED; break;
+                        THREAD_QUEUE.Clear();
+                        ASYNC_THREAD_QUEUE.Clear();
```

Comment:
Should probably move these back to Program as some kinda "clear" function and make non-static so we can hide the queue logic inside Program only